auraed/src/init/network/mod.rs

```diff
@@ -73,7 +73,7 @@ pub(crate) struct Network(Handle);
 impl Network {
     pub(crate) fn connect() -> Result<Network, NetworkError> {
         let (connection, handle, _) = rtnetlink::new_connection()?;
-        tokio::spawn(connection);
+        let _unused_join_handle = tokio::spawn(connection);
```

Comment:
using `let _ =` is fine. You don't need to spend energy naming things that are ignored.